e2e-tests/src/test/finalization.rs

```diff
@@ -1,9 +1,12 @@
-use aleph_client::{create_connection, wait_for_finalized_block};
+use aleph_client::waiting::{AlephWaiting, BlockStatus};
 
 use crate::config::Config;
 
-pub fn finalization(config: &Config) -> anyhow::Result<()> {
-    let connection = create_connection(&config.node);
-    wait_for_finalized_block(&connection, 1)?;
+pub async fn finalization(config: &Config) -> anyhow::Result<()> {
+    let connection = config.create_root_connection().await;
+    connection
+        .connection
+        .wait_for_block(|n| n >= 1, BlockStatus::Finalized)
```

Comment:
I would take the opportunity do an enhanced check:
 - read what is the current block - say `n`
 - wait until `n+1` is finalized

this would increase significance of this test in cases when it is run after some chain operations that might have caused finalization to stall

Reply:
done